### PR TITLE
Update pl-PL.lua

### PR DESCRIPTION
**Pull request by callmeeJeff**

### CODE REVIEW

File: languages/pl-PL.lua [hunk 1]
```diff
@@ -7,8 +7,8 @@ if not OP.Global.Locales.Languages then OP.Global.Locales.Languages = {} end
 
 OP.Global.Locales.Languages["pl-PL"] = {
 	-- configuration settings for language
-	largeNumberSeparator = ",", -- with it set as ",", 1000000 -> 1,000,000 in a lot of labels
-	floatSeparator = ".", -- with it set as ".", (25 + (1/5)) -> 25.2
+	largeNumberSeperator = ",", -- with it set as ",", 1000000 -> 1,000,000 in a lot of labels
+	floatSeperator = ".", -- with it set as ".", (25 + (1/5)) -> 25.2
```
Comment (InZidiuZ) [inline languages/pl-PL.lua:11]: Why are you changing these?

File: languages/pl-PL.lua [hunk 2]
```diff
@@ -590,7 +590,7 @@ OP.Global.Locales.Languages["pl-PL"] = {
 		unscoop_failed = "Nie udało się zebrać graczy.",
 
 		unscoop_missing_permissions = "Gracz próbował zebrać graczy/a bez odpowiednich uprawnień.",
-
+
```
Comment (InZidiuZ) [inline languages/pl-PL.lua:593]: ?

File: languages/pl-PL.lua [hunk 3]
```diff
@@ -687,7 +687,7 @@ OP.Global.Locales.Languages["pl-PL"] = {
 		failed_toggle_strict_mode = "Nie udało się przełączyć trybu ścisłego.",
 		strict_mode_enabled = "Pomyślnie włączono tryb ścisły.",
 		strict_mode_disabled = "Pomyślnie wyłączono tryb ścisły.",
-
+
```
Comment (InZidiuZ) [inline languages/pl-PL.lua:690]: ?

File: languages/pl-PL.lua [hunk 4]
```diff
@@ -724,11 +724,11 @@ OP.Global.Locales.Languages["pl-PL"] = {
 		player_one_label = "Franklin",
 		player_two_label = "Trevor",
 		player_zero_label = "Michael",
-
+
```
Comment (InZidiuZ) [inline languages/pl-PL.lua:727]: ?

File: languages/pl-PL.lua [hunk 5]
```diff
@@ -724,11 +724,11 @@ OP.Global.Locales.Languages["pl-PL"] = {
...
 		player_two_label = "Trevor",
 		player_zero_label = "Michael",
-
+		
 		notification_distance_taze = "Anti-Cheat: ${displayName} ztazerował kogoś na bardzo dużą odległość (${distance}m).",
 		notification_bad_screen_word = "Anti-Cheat: Użycie zabronionego słowa",
-		notification_suspicious_explosion = "Anty-Cheat: ${displayName} wywołał/a eksplozję ${explosionType} (~${distance}m).",
-
+		notification_suspicious_explosion = "Anti-Cheat: ${displayName} wywołał eksplozję typu ${explosionType}.",
+
```
Comment (InZidiuZ) [inline languages/pl-PL.lua:731]: ?

File: languages/pl-PL.lua [hunk 6]
```diff
@@ -772,7 +772,7 @@ OP.Global.Locales.Languages["pl-PL"] = {
 		invincibility_warning = "Zostałeś oflagowany za bycie niewidzialnym! Poinformuj administracje w tickecie i powiedz jej, co robiłeś, że tak się stało, ponieważ nie powinieneś otrzymywać tej wiadomości na czacie.",
 		damage_modifier_warning = "Zostałeś oflagowany za posiadanie zmodyfikowanych obrażeń! Poinformuj administracje w tickecie i powiedz jej, co robiłeś, że tak się stało, ponieważ nie powinieneś otrzymywać tej wiadomości na czacie.",
 		freeze_warning = "Zostałeś oflagowany za bycie zamrożonym! Poinformuj administracje w tickecie i powiedz jej, co robiłeś, że tak się stało, ponieważ nie powinieneś otrzymywać tej wiadomości na czacie.",
-
+
```
Comment (InZidiuZ) [inline languages/pl-PL.lua:775]: ?

File: languages/pl-PL.lua [hunk 7]
```diff
@@ -869,9 +869,9 @@ OP.Global.Locales.Languages["pl-PL"] = {
 
 		command_list = "${commands}",
 		substitute_list = "${substitutes}",
-
+
```
Comment (InZidiuZ) [inline languages/pl-PL.lua:872]: ?

File: languages/pl-PL.lua [hunk 8]
```diff
@@ -869,9 +869,9 @@ OP.Global.Locales.Languages["pl-PL"] = {
 
 		command_list = "${commands}",
 		substitute_list = "${substitutes}",
-
+		
 		substitute_command_for = "To jest polecenie zastępujące `${command}`.",
-
+
```
Comment (InZidiuZ) [inline languages/pl-PL.lua:874]: ?

File: languages/pl-PL.lua [hunk 9]
```diff
@@ -968,7 +968,7 @@ OP.Global.Locales.Languages["pl-PL"] = {
 
 		delete_vehicle_command = "delete_vehicle",
 		delete_vehicle_command_help = "Usuwa najbliższy pojazd .",
-		delete_vehicle_command_parameter_ignore_heading = "ignoruj kierunek",
+		delete_vehicle_command_parameter_ignore_heading = "yes",
```
Comment (InZidiuZ) [inline languages/pl-PL.lua:971]: 'yes'?

File: languages/pl-PL.lua [hunk 10]
```diff
@@ -1107,7 +1107,7 @@ OP.Global.Locales.Languages["pl-PL"] = {
 		disable_recoil_command = "disable_recoil",
 		disable_recoil_command_help = "Wyłącza odrzut wszystkich broni.",
 		disable_recoil_command_substitutes = "",
-
+
```
Comment (InZidiuZ) [inline languages/pl-PL.lua:1110]: ?

File: languages/pl-PL.lua [hunk 11]
```diff
@@ -1327,7 +1327,7 @@ OP.Global.Locales.Languages["pl-PL"] = {
 		drive_speed_command_parameter_speed = "speed",
 		drive_speed_command_parameter_speed_help = "The speed you want to set (in mph).",
 		drive_speed_command_substitutes = "",
-
+
```
Comment (InZidiuZ) [inline languages/pl-PL.lua:1330]: ?

File: languages/pl-PL.lua [hunk 12]
```diff
@@ -1429,7 +1429,7 @@ OP.Global.Locales.Languages["pl-PL"] = {
 		unscoop_command_parameter_revive = "revive",
 		unscoop_command_parameter_revive_help = "Ożyw graczy których zebrałeś.",
 		unscoop_command_substitutes = "",
-
+
```
Comment (InZidiuZ) [inline languages/pl-PL.lua:1432]: ?

File: languages/pl-PL.lua [hunk 13]
```diff
@@ -1497,37 +1497,37 @@ OP.Global.Locales.Languages["pl-PL"] = {
 		emojis_refresh_command = "emojis_refresh",
 		emojis_refresh_command_help = "Refresh the available emojis. This will fetch the latest list from the discord guild.",
 		emojis_refresh_command_substitutes = "",
-
+
```
Comment (InZidiuZ) [inline languages/pl-PL.lua:1500]: ?

File: languages/pl-PL.lua [hunk 14]
```diff
@@ -1551,7 +1551,7 @@ OP.Global.Locales.Languages["pl-PL"] = {
 		tp_here_command_parameter_server_id = "server id",
 		tp_here_command_parameter_server_id_help = "Identyfikator serwera gracza, którego chcesz teleportować.",
 		tp_here_command_substitutes = "",
-
+
```
Comment (InZidiuZ) [inline languages/pl-PL.lua:1554]: ?

File: languages/pl-PL.lua [hunk 15]
```diff
@@ -1593,7 +1593,7 @@ OP.Global.Locales.Languages["pl-PL"] = {
 		show_alert_command_parameter_content = "content",
 		show_alert_command_parameter_content_help = "Treść powiadomienia.",
 		show_alert_command_substitutes = "alert",
-
+
```
Comment (InZidiuZ) [inline languages/pl-PL.lua:1596]: ?

File: languages/pl-PL.lua [hunk 16]
```diff
@@ -1673,7 +1673,7 @@ OP.Global.Locales.Languages["pl-PL"] = {
 		bed_command = "bed",
 		bed_command_help = "Spróbuj położyć się w najbliższym łóżku.",
 		bed_command_substitutes = "",
-
+
```
Comment (InZidiuZ) [inline languages/pl-PL.lua:1676]: ?

File: languages/pl-PL.lua [hunk 17]
```diff
@@ -1712,7 +1712,7 @@ OP.Global.Locales.Languages["pl-PL"] = {
 		stable_cam_command = "stable_cam",
 		stable_cam_command_help = "Przełącza stabilizację kamery.",
 		stable_cam_command_substitutes = "",
-
+
```
Comment (InZidiuZ) [inline languages/pl-PL.lua:1715]: ?

**Review by InZidiuZ — CHANGES_REQUESTED** [review]
Messy PR, please clean up. I got to 17 comments, but I am not going to spend time finding all of the bad parts.

**Comment by callmeeJeff** [comment]
I guess all fixed

**Comment by callmeeJeff** [comment]
Idk what i need to do more

**Review by InZidiuZ — APPROVED** [review]

**Comment by InZidiuZ** [comment]
These will be added with the nest auto-locales.

**Pull request closed without merging**